src/applications/gi/utils/render.jsx

```diff
@@ -3,25 +3,46 @@ import AlertBox from '@department-of-veterans-affairs/formation-react/AlertBox';
 
 export const renderSchoolClosingAlert = result => {
   const { schoolClosing } = result;
+
   if (!schoolClosing) return null;
   return (
     <AlertBox
-      content={<p>Upcoming campus closure</p>}
-      headline="School closure"
+      content={<li>Upcoming campus closure</li>}
```

Comment:
Is switching this to an `li` tag going to cause axe/508 issues?

Reply:
Just ran an Axe analyzation and had one finding, which is related to your comment below about wrapping in a `<ul>`

Reply:
Based on images in story I think you can leave as a `<p>`